spring-xd-messagebus-spi/src/main/java/org/springframework/xd/dirt/integration/bus/MessageBusSupport.java

```diff
@@ -1103,4 +1104,19 @@ public void handleMessage(Message<?> message) throws MessagingException {
 
 	}
 
+	/**
+	 * No op method that can be implemented by specific message bus to store message headers to acknowledge
+	 * the messages manually later.
+	 *
+	 * @param messageHeaders the message headers
+	 */
+	public void storeForManualAck(MessageHeaders messageHeaders) {
+	}
+
+	/**
+	 * Perform manual acknowledgement based on the metadata stored in message bus.
+	 */
+	public void doManualAck() {
+	}
+
```

Comment:
We (Spring) generally don't allow breaking changes to interfaces in point releases - it will break someone's code if, for example, they have implemented a custom MB.

Perhaps unlikely but...